src/main/scala/firrtl/transforms/MustDedup.scala

```diff
@@ -219,25 +217,22 @@ class MustDeduplicateTransform extends Transform with DependencyAPIMigration {
       // Write reports and modules to disk
       val dirName = state.annotations.collectFirst { case MustDeduplicateReportDirectory(dir) => dir }
         .getOrElse("dedup_failures")
-      val dir = new File(dirName)
+      val dir = FileUtils.getPath(dirName)
```

Comment:
I think `getOrElse(xxx)` should be a relative path to `TargetAnnotation`. This should be a legacy issue. I think you can fix this.

Reply:
dedup_failures -> targetDir / "dedup_report"